# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintObject.cpp
# Commit: aea4783f8b48
# Full Hash: aea4783f8b4867167607f8ba00eadc4aaa45d61d
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-02-07 03:55:51
# Regressor Bug: 1582832
# File Overlap Count: 1
# Description:
#   Bug 1582832: Part 3 - Correctly handle windowless BCs created for printing. r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D61006
# ==============================================================================

diff -r cefac6461948 -r aea4783f8b48 layout/printing/nsPrintObject.cpp
--- a/layout/printing/nsPrintObject.cpp	Thu Feb 06 19:08:04 2020 +0000
+++ b/layout/printing/nsPrintObject.cpp	Thu Feb 06 19:08:06 2020 +0000
@@ -77,7 +77,7 @@
     // is detached from any browser window or tab.
 
     // Create a new BrowsingContext to create our DocShell in.
-    RefPtr<BrowsingContext> bc = BrowsingContext::Create(
+    RefPtr<BrowsingContext> bc = BrowsingContext::CreateWindowless(
         /* aParent */ nullptr,
         /* aOpener */ nullptr, EmptyString(),
         aDocShell->ItemType() == nsIDocShellTreeItem::typeContent